exercises/concept/luigis-luscious-lasagna/.docs/hints.md

```diff
@@ -0,0 +1,19 @@
+# General
+
+- The let expressions is described in the [Elm lang documentation][let-elm-lang].
+- Whitespace is important
+- There can only be one expression in the `in` section
```

Comment:
Maybe add a hint that the student can define multiple values in the `let` part by having them on separate lines?

Reply:
Good idea, I've pushed a commit